src/components/UpdateSubscription/Unsubscribe.tsx

```diff
@@ -70,34 +63,22 @@ const Unsubscribe = ({
 
   return (
     <InnerPopupWrapper
-      steps={isFreeExtensionSecondStep ? 2 : steps.length}
+      steps={steps.length}
       popupTitle={t('unsubscribe-popup.title', 'Manage your plan')}
       isError={Boolean(isError)}
-      currentStep={
-        isFreeExtensionSecondStep ? 2 : steps.indexOf(currentStep) + 1
-      }
+      currentStep={steps.indexOf(currentStep) + 1}
     >
-      {currentStep === STEPS.FREE_EXTENSION && (
-        <FreeExtension
-          handleUnsubscribe={goToNextStep}
-          setIsFreeExtensionSecondStep={setIsFreeExtensionSecondStep}
-        />
-      )}
-      {currentStep === STEPS.PAUSE && !!pauseOffer && (
-        <Pause pauseOffer={pauseOffer} handleClick={goToNextStep} />
-      )}
       {currentStep === STEPS.DOWNGRADES && (
         <Downgrades
-          downgrades={downgradesWithoutPause}
+          downgrades={availableDowngrades}
           handleClick={goToNextStep}
         />
       )}
       {currentStep === STEPS.SURVEY && (
         <Survey
           customCancellationReasons={customCancellationReasons}
           checkedReason={checkedReason}
-          shouldShowDowngrades={shouldShowDowngrades}
-          shouldShowFreeExtension={shouldShowFreeExtension}
+          shouldShowDowngrades={steps.length === 3}
```

Comment:
there's an antipattern called `magic numbers`

why `3` specifically? please either extract the number 3 to a const with a name explaining the reasoning

maybe there's a better possible approach? what does the 3 mean here? let's iterate :)

Reply:
fixed :)